<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LATITUDE" failures="1" tests="8" name="Test" time="0.169" errors="0" timestamp="2025-09-21T11:14:21 IST">
  <testcase classname="pack1.InclusionExclusion" name="cestCase4" time="0.02"/>
  <testcase classname="pack1.InclusionExclusion" name="testCase2" time="0.005"/>
  <testcase classname="pack1.InclusionExclusion" name="xestCase1" time="0.008">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at pack1.InclusionExclusion.xestCase1(InclusionExclusion.java:14)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- xestCase1 -->
  <testcase classname="pack1.InclusionExclusion" name="festCase3" time="0.001">
    <skipped/>
  </testcase> <!-- festCase3 -->
  <testcase classname="pack1.InclusionExclusion2" name="cestCase8" time="0.003"/>
  <testcase classname="pack1.InclusionExclusion2" name="festCase7" time="0.003"/>
  <testcase classname="pack1.InclusionExclusion2" name="testCase6" time="0.002"/>
  <testcase classname="pack1.InclusionExclusion2" name="xestCase5" time="0.002"/>
</testsuite> <!-- Test -->
